test/fromFile.test.js

```diff
@@ -9,32 +10,54 @@ const example = require('./support/example')
 describe('fromFile', () => {
   it('should create a quad stream', async () => {
     const stream = fromFile(resolve(__dirname, 'support/example.nt'))
-
     stream.resume()
-
     strictEqual(isReadable(stream), true)
   })
 
   it('should forward options to parser', async () => {
     const stream = fromFile(resolve(__dirname, 'support/example.ttl'), { baseIRI: 'http://example.org/' })
     const dataset = await rdf.dataset().import(stream)
-
     strictEqual(dataset.toCanonical(), example().toCanonical())
   })
 
-  it('should throw an error if the file extension is unknown', () => {
-    throws(() => {
-      fromFile('test.jpg')
-    })
+  it('should throw an error if the file extension is unknown', async () => {
+    await assert.rejects(
+      async () => {
+        fromFile('test.jpg')
```

Comment:
A sync error is thrown and `throws()` should be used. An `await` would be required if an async error is thrown.